scenarios/kubernetes_e2e.py

```diff
@@ -92,17 +92,22 @@ def main(args):
         for env in args.env_file:
             cmd.extend(['--env-file', test_infra(env)])
 
-    gce_ssh = '/workspace/.ssh/google_compute_engine'
-    gce_pub = '%s.pub' % gce_ssh
-    service = '/service-account.json'
+    if args.gce_ssh:
+        gce_ssh = '/workspace/.ssh/google_compute_engine'
+        gce_pub = '%s.pub' % gce_ssh
+        cmd.extend([
+          '-v', '%s:%s:ro' % (args.gce_ssh, gce_ssh),
+          '-v', '%s:%s:ro' % (args.gce_pub, gce_pub),
+          '-e', 'JENKINS_GCE_SSH_PRIVATE_KEY_FILE=%s' % gce_ssh,
+          '-e', 'JENKINS_GCE_SSH_PUBLIC_KEY_FILE=%s' % gce_pub])
+
+    if args.service_account:
+        service = '/service-account.json'
+        cmd.extend([
+            '-v', '%s:%s:ro' % (args.service_account, service),
+            '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service])
```

Comment:
I would actually recommend run it from boostrap.py, do something like
```
./bootstrap.py --job=FOO --bare --json=1 --timeout=X --service-account=/path/to/cred
```
and it should set the rest up?